examples/meta/src/evaluation/cross_validation_support_vector_machine.sg

```diff
@@ -8,11 +8,11 @@ Machine svm = machine("LibLinear")
 svm.put("liblinear_solver_type", enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
 
 StratifiedCrossValidationSplitting splitting_strategy(labels, 5)
-AccuracyMeasure evaluation_criterion()
+Evaluation evaluation_criterion = evaluation("AccuracyMeasure")
 CrossValidation cross(svm, feats, labels, splitting_strategy, evaluation_criterion)
 cross.set_num_runs(2)
 
-CrossValidationResult result = CrossValidationResult:obtain_from_generic(cross.evaluate())
+CrossValidationResult result = cross.evaluate()
```

Comment:
this assumes that one could explicitly cast `CEvaluationResult` to `CrossValidationResult`, which is not the case unfortunately.

for the time being there's no solution for this other than the one you had before:
```
CrossValidationResult result = CrossValidationResult:obtain_from_generic(cross.evaluate())
```

Reply:
`EvalidationResult result = cross.evaluate()`  should work